changelog.d/20231031_230517_yadudoc1729_task_retries_1.rst

```diff
@@ -0,0 +1,12 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- ``GlobusComputeEngine`` can now be configured to automatically retry task failures when
+  node failures occur. Traceback history from all prior attempts is supplied if the last retry
+  attempt fails. Here's a snippet from config.yaml:
+
+  ```
+  engine:
+    type: GlobusComputeEngine
+    max_retries_on_system_failure: 2
+  ```
```

Comment:
Reminder that this is reST, not Markdown.  The syntax is `.. code-block::`:

```reST
.. code-block:: yaml
  engine:
    type: GlobusComputeEngine
    max_retries_on_system_failure: 2
```

Reply:
Fixed